plugins/airtable/src/SelectDataSource.tsx

```diff
@@ -96,47 +116,58 @@ export function SelectDataSource({ collection, onSelectDataSource }: SelectDataS
                 <img src="airtable.svg" alt="Airtable icon" style={{ width: 80, height: 80 }} />
             </div>
 
-            <label htmlFor="base">
-                Base
-                <select
-                    id="base"
-                    onChange={event => {
-                        setSelectedBaseId(event.target.value)
-                    }}
-                    value={selectedBaseId}
-                    disabled={status === "loading-bases"}
-                >
-                    <option value="" disabled>
-                        {status === "loading-bases" ? "Loading…" : "Choose…"}
-                    </option>
-                    {bases.map(({ id, name }) => (
-                        <option key={id} value={id}>
-                            {name}
+            <div className="setup-list">
+                <label htmlFor="base">
+                    Base
+                    <select
+                        id="base"
+                        onChange={event => {
+                            setSelectedBaseId(event.target.value)
+                        }}
+                        value={selectedBaseId}
+                        disabled={status === "loading-bases" || status === "error-bases"}
+                    >
+                        <option value="" disabled>
+                            {status === "loading-bases" ? "Loading…" : status === "error-bases" ? "Error" : "Choose…"}
```

Comment:
As in the previous case, try to avoid nested ternary expressions for readability.